src/main/java/novamachina/exnihilosequentia/ExNihiloSequentia.java

```diff
@@ -69,36 +51,5 @@ public static void registerModifierSerializers(
                     .register(new UseCrookModifier.Serializer()
                             .setRegistryName(ExNihiloConstants.ModIds.EX_NIHILO_SEQUENTIA, "use_crook"));
         }
-
-        public static void onBlockBreak(BlockEvent.BreakEvent event) {
-            if (!event.getWorld().isClientSide() && !event.getPlayer().isCreative() && event.getState().getBlock() == Blocks.STONE) {
-                ItemStack held = event.getPlayer().getMainHandItem();
-                if (!(event.getPlayer() instanceof FakePlayer) && held.isEmpty()) {
-                    int j = new Random().nextInt(3);
-                    ItemStack stack = EnumPebbleType.STONE.getRegistryObject().get().getDefaultInstance();
-                    for (int i = 0; i <= j; i++) {
-                        event.getWorld().addFreshEntity(new ItemEntity((World) event.getWorld(), event.getPos().getX() + .5, event.getPos().getY() + .5, event.getPos().getZ() + .5, stack));
-                    }
-                } else if (!(event.getPlayer() instanceof FakePlayer) && held.getItem() instanceof CrookBaseItem) {
-                    int j = new Random().nextInt(3) + 3;
-                    ItemStack stack = EnumPebbleType.STONE.getRegistryObject().get().getDefaultInstance();
-                    for (int i = 0; i <= j; i++) {
-                        event.getWorld().addFreshEntity(new ItemEntity((World) event.getWorld(), event.getPos().getX() + .5, event.getPos().getY() + .5, event.getPos().getZ() + .5, stack));
-                    }
-                }
-            }
-        }
-
-        public static void blockBreakSpeed(PlayerEvent.BreakSpeed event) {
-            if (event.getState().getBlock() == Blocks.STONE) {
-                double speed = 5D;
-                if (event.getPlayer().getMainHandItem().isEmpty()) {
-                    event.setNewSpeed((float) speed);
-                } else if (event.getPlayer().getMainHandItem().getItem() instanceof CrookBaseItem) {
-                    speed = speed * 3;
-                    event.setNewSpeed((float) speed);
-                }
-            }
-        }
```

Comment:
What is the reason that this was removed?

Reply:
will be moved to own mod.